app/uk/gov/hmrc/merchandiseinbaggage/service/Auditor.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.merchandiseinbaggage.service
+
+import play.api.Logger
+import play.api.i18n.{Lang, Messages, MessagesApi, MessagesImpl}
+import play.api.libs.json.Json.toJson
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.merchandiseinbaggage.model.api.Declaration
+//import uk.gov.hmrc.merchandiseinbaggage.model.audit.RefundableDeclaration
+import uk.gov.hmrc.play.audit.http.connector.AuditConnector
+import uk.gov.hmrc.play.audit.http.connector.AuditResult.{Disabled, Failure, Success}
+import uk.gov.hmrc.play.audit.model.ExtendedDataEvent
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.control.NonFatal
+
+trait Auditor {
+  val auditConnector: AuditConnector
+  val messagesApi: MessagesApi
+
+  val messagesEN: Messages = MessagesImpl(Lang("en"), messagesApi)
```

Comment:
please double check if we really need this  messagesEN

Reply:
deleted